sirepo/package_data/static/json/genesis-schema.json

```diff
@@ -0,0 +1,1295 @@
+            ],
+            "ndcut": [
+                "NDCUT",
+                "Integer",
+                0,
+                "When loading a slice, only particles of the external distribution are used, which falls within a small time-window centered around the current position of the slice. If NDCUT has a value larger than zero the width is calculated by (tmax-tmin)/NDCUT, where tmax and tminare determined, while scanning through the particle distribution. If NDCUT is zero, the time-window is adjusted, so that in average NPART/NBINS particles fall in each slice."
+            ],
+            "offsetradf": [
+                "OFFSETRADF",
+                "Integer",
+                0,
+                "If the automatic alignment of the radiation field is disabled by setting ALIGNRADF to a non-zero value, the default alignment is that the first slice of the radiaiton field overlaps with the first slice of the electron beam. However the relative position can be controlled by OFFSETRADF. The value of OFFSETRADF defines the number of slice to skip before filling it for the first electorn slice. E.g. a value of 4 will uses the 5th slice for the simulation of the first slice. slices one to 4 will be used to fill up the slippage field. If Genesis 1.3. require to fill a slice which is not defined by the FIELDFILE then it uses the internal method of a fundamental Gauss-Hermite mode."
+            ],
+            "outputfile": [
+                "OUTPUTFILE",
+                "String",
+                "genesis.out",
+                "The name of the main output file. The prompt for the output filename at the beginning of a GENESIS 1.3 run will be suppressed."
+            ],
+            "partfile": [
+                "PARTFILE",
+                "String",
+                "",
+                "Defines the file containing the particle distribution. The distribution is directly imported into the arrays holding the particle variables. "
+            ],
+            "radfile": [
+                "RADFILE",
+                "String",
+                "",
+                "Specifying a file containing a lookup table for the seeding radiation pulse  at different position along the bunch. "
+            ],
+            "trama": [
+                "TRAMA",
+                "Integer",
+                0,
+                "Non zero value enables that a transport matrix is applied to the electron distribution when importing it with PARTFILE. The individual matrix is defined by ITRAM##"
+            ]
+        },
+	"parameterAnimation": {
```

Comment:
If you want an easy way to add a log scale option, add a "plotScale" field to the model and the standard plotting should pick it up. See srw-schema.json brillianceReport as an example.

Reply:
Also, we usually include a "notes" field on all reports.